### PR TITLE
add usage for cycle-fela

### PR DESCRIPTION
**Pull request by wcastand**
I still need to update my package first to handle createComponent composition.

I probably add global Theming too, so i'll update the doc accordingly.

### CODE REVIEW

**Comment by rofrischmann** [comment]
But is it ready to be merged already ;)?

**Comment by wcastand** [comment]
No, i don't know how to make a WIP PR in github :/
But i'll need a review because my english is far from perfect, so i made the PR now someone have the time to look at. 

Sorry, i forgot to say that :/

**Comment by rofrischmann** [comment]
Hehe no problem. You actually can't do that I guess. Simply add me as a reviewer once you're done :)

**Comment by wcastand** [comment]
Okay, i didn't had the global Theming for now. But the composition in createComponent is shipped :)

**Comment by wcastand** [comment]
I think it should be fine, just need a english review.
I'll make an other PR for displayName recipes once this one is okay :)

File: docs/guides/UsageWithCycle.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+```
+
+### Setup fela in cycle
+
+To initiate cycle with fela, you need to :
+```javascript
+import {makeFelaDomDriver} from 'cycle-fela'
+```
+
+`makeFelaDomDriver` should replace your `makeDomDriver` from `@cycle/dom`. A really besic setup would looks like this:
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: basic* :P

File: docs/guides/UsageWithCycle.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+# Usage with Cycle
+
+Fela was always designed with React in mind, but is **not** bound to React by default. If you want to use it with Cycle, you should also install the unofficial [Cycle bindings for Fela](https://github.com/wcastand/cycle-fela).
+
+```sh
+npm i --save cycle-fela
+```
+
+### Setup fela in cycle
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: Fela in Cycle (for consistence)

File: docs/guides/UsageWithCycle.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+run(main, { DOM: makeFelaDomDriver('#app') })
+
+```
+
+`makeFelaDomDriver` takes optional parameters.
+
+An options object which let you pass the fela renderer options like plugins, enhancers.
+You can also use it to pass a custom DOMElement for your style with `customStyleNode` and your theme for fela.
+
+the third parameter is an array of the [staticRules](http://fela.js.org/docs/api/fela/Renderer.html#renderstaticstyle-reference) for fela.
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: of staticRules

File: docs/guides/UsageWithCycle.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+You can find more informations about options for `makeFelaDomDriver` [here](https://github.com/wcastand/cycle-fela#makefeladomdriver)
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: about the options

File: docs/guides/UsageWithCycle.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+#### createComponent
+
+Cycle-fela exposes `createComponent` like react-fela.
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: cycle-fela

File: docs/guides/UsageWithCycle.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+[createComponent](https://github.com/wcastand/cycle-fela#createcomponent) help you create presentational components that can be reuse in your app.
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: reused

File: docs/guides/UsageWithCycle.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+  return { DOM: vdom$ }
+}
+
+run(main, { DOM: makeFelaDomDriver('#app') })
+
+```
+
+## Composition
+
+For composition, you can use `combineRules` from fela like this :
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: you can use `combineRules` e.g.

File: docs/guides/UsageWithCycle.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+  const RedSpan = createComponent(() => ({color: 'red' }), 'span');
+  const Bold = () => ({fontWeight: 'bold'});
+
+  const BoldRedSpan = createComponent(Bold, RedSpan);
+  export default BoldRedSpan;
+```
+
+## Theming
+
+Like `react-fela` we provide a way to pass a theme to your components.
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: react-fela, we

File: docs/guides/UsageWithCycle.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+
+```javascript
+
+const theme = { primaryColor: 'red' }
+
+makeFelaDomDriver('#app', { theme })
+
+```
+
+## More informations
```
Comment (rofrischmann) [inline docs/guides/UsageWithCycle.md]: information (there's no plural :P)

**Comment by rofrischmann** [comment]
Also would you add your cycle-fela example to the README (Examples)?

File: docs/guides/UsageWithCycle.md (hunk 7 above)
Comment (wcastand) [inline docs/guides/UsageWithCycle.md]: i'm not sure i understand this one :/

**Comment by wcastand** [comment]
It should be all good for me :)

**Pull request merged**